src/home/getters/get-github-access-token.ts

```diff
@@ -19,6 +22,17 @@ export function getGitHubAccessToken(): string | null {
   return null;
 }
 
+export function getGitHubUserName(): string {
+  const oauthToken = getLocalStore(`sb-${SUPABASE_STORAGE_KEY}-auth-token`) as OAuthToken | null;
+
+  const username = oauthToken?.user?.user_metadata?.user_name;
+  if (username) {
+    return username;
+  }
+
+  return "";
```

Comment:
```suggestion
  return null;
```

Returning empty strings leads to a lot of redundant logic to check it later. Something falsy, like null, is best.